Tools/WinMLRunner/src/Run.cpp

```diff
@@ -295,18 +283,34 @@ HRESULT EvaluateModel(
 
         if (FAILED(bindInputResult))
         {
+            output.PrintBindingInfo(i + 1, deviceType, inputBindingType, inputDataType, deviceCreationLocation, "[FAILED]");
             return bindInputResult;
         }
-
-        output.PrintEvaluatingInfo(i + 1, deviceType, inputBindingType, inputDataType, deviceCreationLocation);
+        else if (i == 0)
```

Comment:
Hey @pmbrown1055  would it be possible to keep default behavior for non performance scenarios? By starting to separate here, it would help in the future for refactoring between performance / validation scenarios.
```suggestion
        if (!args.PerfCapture()) //When not capturing performance, we want to keep default behavior.
        {
                output.PrintBindingInfo(i + 1, deviceType, inputBindingType, inputDataType, deviceCreationLocation, "[SUCCESS]"); 
        }
        else if (i == 0) //If capturing performance, we want to decrease amount of printing to console
        {
            output.PrintBindingInfo(i + 1, deviceType, inputBindingType, inputDataType, deviceCreationLocation, "[SUCCESS]");
        }
```

Reply:
My intention was to clean up the console output for all scenarios... but if you think it's important to keep all the output for all iterations for the default case, then I can leave it in.  If that is the case, then we should probably add some kind of -terse or -silent option back in to all the user to not see the default prints...

Reply:
Okay I think that works! How about this for adding the -terse option?:

```suggestion
        if (!args.PerfCapture() && !args.TerseOutput()) //When not capturing performance and the output isn't set to terse, we want to keep printing out binding information.
        {
                output.PrintBindingInfo(i + 1, deviceType, inputBindingType, inputDataType, deviceCreationLocation, "[SUCCESS]"); 
        }
        else if (i == 0) //If capturing performance, we want to decrease amount of printing to console
        {
            output.PrintBindingInfo(i + 1, deviceType, inputBindingType, inputDataType, deviceCreationLocation, "[SUCCESS]");
        }
```